<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileWidget</class>
 <widget class="QWidget" name="FileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件系统</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="m_showList"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="m_return">
       <property name="text">
        <string>返回上一级</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_createDir">
       <property name="text">
        <string>创建文件夹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_deleteDir">
       <property name="text">
        <string>删除文件夹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_renameDir">
       <property name="text">
        <string>重命名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_flushFile">
       <property name="text">
        <string>刷新文件夹</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="m_uploadFile">
       <property name="text">
        <string>上传文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_downloadFile">
       <property name="text">
        <string>下载文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_deleteFile">
       <property name="text">
        <string>删除文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_shareFile">
       <property name="text">
        <string>分享文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_moveFile">
       <property name="text">
        <string>移动文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_selectFile">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>目标目录</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
